modules/calendar/calendar.php

```diff
@@ -1267,6 +1267,9 @@ public function generate_post_li_html($post, $post_date, $num = 0, $show_posts_p
             if ($show_posts_publish_time) {
                 $post_publish_datetime = get_the_date('c', $post);
                 $post_publish_date_timestamp = strtotime($post_publish_datetime);
+                $posts_publish_time_format = is_null($this->module->options->posts_publish_time_format)
+                    ? 'ha'
```

Comment:
The default isn't 'ga', instead of 'ha'? This is why I think we need a constant for the default time format.

Reply:
Sure.